__mocks__/uuid.js

```diff
@@ -1 +1,6 @@
-module.exports = { v1: jest.fn(() => 1) };
+module.exports = {
+  v1: () => {
+    let value = 0;
+    return value++;
```

Comment:
I don't think this is working, all ids resolve to `0`. I don't think it's worth changing the value of the id anyway?